### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by Lucas-Erkana**
✅ Add JavaScript classes instead of objects and arrays.
✅ Add objects and arrays inside classes.
✅ Create class methods to add and remove books
✅ Maintain all of the original functionality of the application
✅ Apply styles (colors, fonts, backgrounds, etc.), 
✅ Pass all linters

### CODE REVIEW

File: module/store.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  static removeBook(author) {
+    const books = Store.getBooks();
+    books.forEach((book, index) => {
+      if (book.author === author) {
+        books.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(books));
```
Comment (brytebee) [inline module/store.js:26]: - After removing books, they're not correctly saved to the local storage.

not_saved

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi Team 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: module/store.js (hunk 1 above)
Comment (Lucas-Erkana) [reply module/store.js:26]: Fixed this issue at this [line](https://github.com/Lucash2022/Awesome_Book/blob/009eca8b70ea7e8797c2354c5c6d960e4b5594b4/main.js#L19)

**Review by andgarzonmal — APPROVED** [review]
Hi @Lucash2022, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**